Assets/Scripts/Audio/AudioController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+
+public class AudioController : MonoBehaviour {
```

Comment:
Can we change the name of this?
Audio Controller is way to broad of a name. Also I believe I have created something similar in name.
Maybe SoundEffectFadeController or something of that sort.